SnapshotTests/extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,66 @@
+import UIKit
+import SnapshotTesting
+
+extension UIViewController {
+    func testSnapshot(
+        as mode: SnapshotMode,
+        in orientation: SnapshotOrientation,
+        named name: String? = nil,
+        record recording: Bool = false,
+        file: StaticString = #file,
+        functionName: String = #function,
+        line: UInt = #line
+    ) {
+        let snapshotting: Snapshotting<UIViewController, UIImage>
+        switch mode {
+        case .accessibilityImage(let precision):
+            snapshotting = .accessibilityImage(precision: precision)
+        case .image:
+            snapshotting = orientation == .portrait ? .image : .imageLandscape
+        case .extra3LargeFont:
+            snapshotting = orientation == .portrait ? .extra3LargeFontStrategy : .extra3LargeFontStrategyLandscape
```

Comment:
What is the `precision` value for `.image` and `.extra3LargeFont` modes?

Reply:
I believe it is 1.0 by default

Reply:
Since our custom precision has move up to 1.0 too, we can remove that from the associated value